docs/web/concepts-layout.mdx

```diff
@@ -0,0 +1,303 @@
+
+Slots control how one element fits into another. But how do we control how sibling elements position themselves? The answer is stacks and splits.
+
+## Stacks and Splits
+
+The basic `stack` and `split` components form building blocks that layout authors can use to vertically or horizontally partition components and blocks of rich text to form simple and complex user interfaces alike.
+
+Leveraging these simple primitives correctly also means responsive apps that work on various screen sizes.
+
+Stacks and splits are used to partition the viewer into rectangular sectors, each filled with whichever components you desire.
+
+The best way to think of stacks is that every item within them is being ordered top to bottom:
+
+```xml
+<stack>
+    <text text="item1"/>
+    <text text="item2"/>
+    <text text="item3"/>
+</stack>
+```
+
+<img className="stackSplitImage" src={useBaseUrl("img/stack-split-1.png")} />
+
+The best way to think of splits is that every item within them is being ordered left to right:
+
+```xml
+<split>
+    <text text="item1"/>
+    <text text="item2"/>
+    <text text="item3"/>
+</split>
+```
+
+<img className="stackSplitImage" src={useBaseUrl("img/stack-split-2.png")} />
+
+When creating complex layouts with embedded stacks and splits, it's good to picture building it from the inside out:
+
+```xml
+<stack>
+    <split>
+        <text text="item1"/>
+        <stack>
+            <text text="item2"/>
+            <text text="item3"/>
+        </stack>
+    </split>
+    <split>
+        <text text="item4"/>
+        <text text="item5"/>
+        <text text="item6"/>
+    </split>
+</stack>
+```
+
+We start by building the first inner-most stack:
+
+<img className="stackSplitImage" src={useBaseUrl("img/stack-split-4.png")} />
+
+Then slot that into the split one level up:
+
+<img className="stackSplitImage" src={useBaseUrl("img/stack-split-5.png")} />
+
+We then build the next split:
+
+<img className="stackSplitImage" src={useBaseUrl("img/stack-split-6.png")} />
+
+And finally slot those two splits into the outer-most stack:
+
+<img className="stackSplitImage" src={useBaseUrl("img/stack-split-3.png")} />
+
+## Panels
+
+Another core concept that Geocortex Web uses to organize elements in a layout is **panels**.
+Panels are a container component that enable hierarchal navigation between different components.
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<layout xmlns="https://geocortex.com/layout/v1">
+    <split resizable="true">
+        <panel width="26">
+            <stack>
+                <search/>
+                <results-list/>
+            </stack>
+            <feature-details active="false"/>
+        </panel>
+        <map id="myMap">
+        </map>
+    </split>
+</layout>
+```
+
+When you first open the panel, the first component in the panel that is not explicity set to `active="false"` is activated. In this example, it's the `<stack>` component containing a `<search>` and `<results-list>`.
```

Comment:
@eric-parsons  I totally agree that activation is an important concept for creating interactive applications! I have to admit though, I know very little about activation in web. Would you be willing to write that article possibly in a future PR, as I feel it might be a lot simpler than you explaining it for me to paraphrase? It's a great idea, and I'm gonna throw it in a ticket in our backlog for sure for now.

Reply:
Sure, I can take that on at some future point. I won't be able to do that in time for the preview release, though. In the meantime do you agree that mentioning `active=` here can just be omitted?

Reply:
yes I totally agree! I have omitted it from here.